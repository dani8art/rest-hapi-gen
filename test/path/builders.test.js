const builders = require('../../lib/path/builders');

describe('Path Builders Tests', () => {
  const basePath = '/tests/base/path';
  const collectionName = 'tests';

  it('#singularPathBuilder should return the proper path', () => {
    const actualPath = builders.singularPathBuilder(basePath, collectionName);

    expect(actualPath).toMatchSnapshot();
  });

  it('#pluralPathBuilder should return the proper path', () => {
    const actualPath = builders.pluralPathBuilder(basePath, collectionName);

    expect(actualPath).toMatchSnapshot();
  });
});
