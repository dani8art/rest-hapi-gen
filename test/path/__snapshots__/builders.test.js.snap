// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Path Builders Tests #pluralPathBuilder should return the proper path 1`] = `"/tests/base/path/tests"`;

exports[`Path Builders Tests #singularPathBuilder should return the proper path 1`] = `"/tests/base/path/tests/{identifier}"`;
